src/commands/commander.h

```diff
@@ -126,7 +126,7 @@ struct CommandAttributes {
   // negative number -n means number of arguments is equal to or large than n
   int arity;
 
-  // space-splitted flag strings to initialize flags
+  // space-sparated flag strings to initialize flags
```

Comment:
I think there is another typo: `separated`.

Reply:
```suggestion
  // space-separated flag strings to initialize flags
```

🤣

Reply:
Report to upstream dict - https://github.com/crate-ci/typos/issues/784#issuecomment-1673403677